go.mod

```diff
@@ -21,12 +21,12 @@ require (
 	github.com/kubernetes-csi/external-snapshotter/v2 v2.2.0-rc1
 	github.com/onsi/ginkgo v1.13.0
 	github.com/onsi/gomega v1.10.1
-	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v1.0.0
+	github.com/pkg/errors v0.9.1
```

Comment:
Was the bump on this version required by another dependency?

Reply:
Yes, the cluster/api uses that version, and we are using the cluster/api `.Patch` function:

https://github.com/vmware-tanzu/velero/pull/2870/files#diff-37aff102a57d3d7b797f152915a6dc16R40

https://github.com/kubernetes-sigs/cluster-api/blob/master/go.mod#L23